docs/src/API.md

```diff
@@ -126,3 +127,11 @@ CommonTypes.DesertDustType
 PrecipitationSusceptibility.precipitation_susceptibility_autoconversion
 PrecipitationSusceptibility.precipitation_susceptibility_accretion
 ```
+
+# Terminal Velocity (Chen 2022)
+
+```@docs
+TerminalVelocityChen2022
```

Comment:
Now that we moved the velocities to `Microphysics2M` module, we should delete all of that.